integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/sortcolumns/TestSortColumns.scala

```diff
@@ -447,7 +447,7 @@ class TestSortColumns extends QueryTest with BeforeAndAfterAll {
   def setLoadingProperties(offheap: String, unsafe: String, useBatch: String): Unit = {
     CarbonProperties.getInstance().addProperty(CarbonCommonConstants.ENABLE_OFFHEAP_SORT, offheap)
     if (useBatch.equalsIgnoreCase("true")) {
```

Comment:
use `unsafe.equalsIgnoreCase("true")` instead of `useBatch.equalsIgnoreCase("true")`